README.md

```diff
@@ -1 +1,59 @@
-# kanban-2
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Space Traveler
+
+> A web application for a company that provides commercial and scientific space travel services. The application will allow users to book rockets and join selected space missions.
+
+## Built With
+
+- React
+- Redux
+
+## Live Demo
+
+[Live Demo Link](https://github.com/Joel-100/kanban-2)
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+- Clone the repo to your local environment
+- Run `npm install` to install all dependencies
+- Run `npm run start` to open it in your browser.
+- Modify to your taste with your preferred text editor
+- Enjoy 😊
+
+## Authors
+
+👤 **Joel Kugonza**
+
+- GitHub: [@joelkugonza](https://github.com/Joel-100)
+- Twitter: [@joelkugonza](https://twitter.com/JoelKugonza)
+- LinkedIn: [@joelkugonza](https://www.linkedin.com/in/joel-kugonza-879211190/)
+
+👤 **Freddy Kaberuka**
+
+- GitHub: [@freddykaberuka](https://github.com/freddykaberuka)
+- Twitter: [@trojanx07](https://twitter.com/trojanx07)
+- LinkedIn: [@freddykaberuka](https://www.linkedin.com/in/kaberuka-freddy-853b08153/)
+
+
```

Comment:
- [ ] Kindly add the "future features" session to your README. Makes the information provided on your README more professional. You can use the following [template](https://github.com/microverseinc/readme-template).